lstchain/image/muon/muon_analysis.py

```diff
@@ -247,12 +292,18 @@ def analyze_muon_event(event_id, image, geom, equivalent_focal_length,
         plot_muon_event(ax, geom, image * clean_mask, centroid, ringrad_camcoord, 
                         ringrad_inner, ringrad_outer, event_id)
 
+        plt.figtext(0.15, 0.20, 'radial std dev: {0:.3f}'.format(radial_distribution['standard_dev']))
```

Comment:
should this additional code go directlly into the plotting function?

Reply:
Not sure it is nicer, it means passing three additional arguments.